packages/core/src/utils/helper.ts

```diff
@@ -201,4 +201,20 @@ export function checkMFPlugin(config: LibConfig): boolean {
   return added;
 }
 
+export function composeMFPlugin(config: LibConfig): RsbuildConfig {
+  if (config.format !== 'mf' || process.env.NODE_ENV !== 'production') {
```

Comment:
`rslib build` and `rslib build --watch` all uses production mode by default, so the default moduleIds should be `deterministic` by default. Why we do this?

Reply:
Got it, so ``process.env.NODE_ENV !== 'production'` is used to distinguish `mf dev`